avocado/core/virt.py

```diff
@@ -16,16 +16,13 @@
 Module to provide classes for Virtual Machines.
 """
 
-import logging
 import time
 from xml.dom import minidom
 
 import libvirt
 
 from . import remoter
 
-LOG = logging.getLogger('avocado.test')
```

Comment:
Well this should be squashed to previous commit as that is where the LOG usage is removed...